.github/workflows/test-integration.yml

```diff
@@ -0,0 +1,31 @@
+name: test-integration
+on:
+  pull_request:
+  push:
+    branches: [ master, develop ]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Get Composer Cache Directory
+        id: composer-cache
+        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+      - uses: actions/cache@v2
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
+          restore-keys: ${{ runner.os }}-composer-
+      - name: Init environment
+        run: cd ci/docker && docker-compose up -d
+      - name: Install dependencies
+        run: cd ci/docker && docker-compose exec -T tiqr-server-libphp bash -lc 'composer install'
+      - name: Run Static analysis
+        run: cd ci/docker && docker-compose exec -T tiqr-server-libphp bash -lc 'composer static-analysis'
+      - name: Run test scripts
+        run: cd ci/docker && docker-compose exec -T tiqr-server-libphp bash -lc 'composer phpunit && composer security-tests'
```

Comment:
Why not create a `test` composer script, and run the required tests from there? That's how we do it in the other projects right? I'm also fine with keeping it this way. It's easy enough to restructure when the need arises.

Reply:
As explained in person, Currently the Static analysis tests are allowed to fail due to the code being legacy code. However the Static analysis gives a lot of style warnings and errors. Therefore i felt that it was a good idea to split these tests up otherwise you would not be able to read the outputs of the other tests easily.

Reply:
Good suggestion!